docs/lazy-adr/adr-005-decouple-blockid-and-partsetheader.md

```diff
@@ -0,0 +1,49 @@
+# ADR 005: Decouple the `PartSetHeader` from the `BlockID`
+
+## Changelog
+
+- 2021-08-01: Initial Draft
+
+## Context
+
+Celestia has multiple commits to the block data via the `DataHash` and the `PartSetHeader` in the `BlockID`. As stated in the #184, we no longer need the `PartSetHeader` for this additional commitment to the block’s data. However, in the short-medium term, we are still planning on using the `PartSetHeader` to be used for block propagation during consensus. This means that we will remove the `PartSetHeader` from as many places as possible, but keep it in the `Proposal` struct.
+
+## Alternative Approaches
+
+It’s worth noting that there are proposed changes to remove the `PartSetHeader` entirely, and instead use the already existing commitment to block data, the `DataAvailabilityHeader`, to propagate blocks in parallel during consensus. Discussions regarding the detailed differences entailed in each approach are documented in that ADR’s PR. The current direction that is described in this ADR is significantly more conservative in its approach, but it is not strictly an alternative to other designs. This is because other designs would also require removal of the `PartSethHeader`, which is a project in and of itself due to the `BlockID` widespread usage throughout tendermint and the bugs that pop up when attempting to remove it. 
+
+## Decision
+
+While we build other better designs to experiment with, we will continue to implement the design specified here as it is not orthogonol. https://github.com/celestiaorg/lazyledger-core/pull/434#issuecomment-869158788
+
+## Detailed Design
+
+- [X] Decouple the BlockID and the PartSetHeader [#441](https://github.com/celestiaorg/lazyledger-core/pull/441)
+- [ ] Remove the BlockID from every possible struct other than the `Proposal`
+  - [X] Stop signing over the `PartSetHeader` while voting [#457](https://github.com/celestiaorg/lazyledger-core/pull/457)
+  - [X] Remove the `PartSetHeader` from the Header [#457](https://github.com/celestiaorg/lazyledger-core/pull/457)
+  - [X] Remove the `PartSetHeader` from VoteSetBits, VoteSetMaj23, and `state.State` [#479](https://github.com/celestiaorg/lazyledger-core/pull/479)
+  - [ ] Remove the `PartSetHeader` from other structs
+
+
+## Status
+
+{Accepted}
```

Comment:
![image](https://user-images.githubusercontent.com/3290375/127792108-5ada9cb8-62d5-4e2e-b782-5f26657567b9.png)

Reply:
sorry, I assumed that it was already accepted as we've already merged the implementation to master

https://github.com/celestiaorg/lazyledger-core/pull/441#pullrequestreview-709054968

updated in https://github.com/celestiaorg/lazyledger-core/pull/482/commits/49315e6b087213e4263eafb7dd273a2bd80c1ef5

Reply:
Just joking 😂